src/elements/content-explorer/ContentExplorer.js

```diff
@@ -357,15 +362,13 @@ class ContentExplorer extends Component<Props, State> {
             view: VIEW_METADATA,
         });
 
-        // Fetch the Metadata Query Results
-        this.api.getMetadataQueryAPI().queryMetadata(
+        const metadataQueryAPIHelper = new MetadataQueryAPIHelper(
```

Comment:
Should this helper live as an instance field on `ContentExplorer`? 

Would this `MetadataQueryAPIHelper` be used for updating the metadata as well? Should the `MetadataQueryAPIHelper` class be structured to be more general? It seems like passing in the query and callbacks into the constructor lend it to serve a very specific purpose. 

Also, is there a naming convention we want to follow for these API helper classes?